content/lessons/layouts/layout-basics.md

```diff
@@ -0,0 +1,114 @@
+---
+title: Layout Basics
+template: lesson
+slug: /courses/Layouts/layout-basics
+course: Layouts
+defaultTab: tests
+tags:
+  - Refactoring
+description: In this lesson you will be introduced to concepts such as the box model, positioning, and floats.
+timeToCompletion: ~1 hour
+videoLinks: 
+ - https://www.youtube.com/embed/rIO5326FgPE
```

Comment:
I've found it's helpful to balance videos and reading material. I'd recommend having a balance here, versus making this all video, which might cause some people to become lost.